src/idaplugin/defs.cpp

```diff
@@ -44,6 +45,10 @@ int runCommand(
 		}
 		return 1;
 	}
+	*hdl = p;
```

Comment:
How is it possible that this does not crash for you? `runCommand()` is also used in `RdGlobalInfo::initPythonCommand()` in `defs.cpp` module - this time with default parameters - i.e. `hdl` is not set. Setting it here when `hdl` is `nullptr` crashes for me. I think there should be something like this:
```c
if (hdl)
{
	*hdl = p;
}
```
Or the `hdl` can not have a default value set and `runCommand()` uses in `RdGlobalInfo::initPythonCommand()` need to change.

You likely have Python interpreter explicitly set so that the code causing problems in `RdGlobalInfo::initPythonCommand()` does not get executed?

Reply:
Yes I did not test it when the pointer was null - but absolutely it needs to be guarded.  Thanks for this as well.